ref/feature/openidConnectServer-1.0.adoc

```diff
@@ -1,12 +1,17 @@
 = OpenID Connect Provider 1.0
-:linkcss: 
+:linkcss:
 :page-layout: feature
-:nofooter: 
+:nofooter:
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=description]
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=enable]
 
+<<<<<<< Updated upstream
```

Comment:
Don't think these git diff lines should still be in the file.

Reply:
Removed.

Reply:
Removed